src/plugins/initiatives/initiativesMap.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+
+import {type Compatible, fromCompat, toCompat} from "../../util/compat";
+import {type URL} from "./initiative";
+
+const COMPAT_INFO = {type: "sourcecred/initiatives", version: "0.1.0"};
+
+/**
+ * JSON (compat) representation of an Initiative.
+ *
+ * Major difference is that instead of using tracker addresses, we use object
+ * keys as an ID, and derive the tracker address from that.
+ */
+export type InitiativesMap = {[entryKey: string]: InitiativeEntry};
+
+type InitiativeEntry = {|
+  +title: string,
+  +timestampMs: number,
+  +completed: boolean,
+  +dependencies: $ReadOnlyArray<URL>,
+  +references: $ReadOnlyArray<URL>,
+  +contributions: $ReadOnlyArray<URL>,
```

Comment:
I suspect we have more edge types than we need here -- it's not super clear, for example, when something is a dependency vs a contribution. Since it's easier to add complexity than to remove it, I propose that we remove the dependency type until we're sure what we need it for.

Reply:
These types are compatible with the main `Initiative` type, which we will need for conversion.
https://github.com/sourcecred/sourcecred/pull/1417/files

We can remove these in it's own PR, though I feel that would be a loss of fidelity.
Because the directional flows of these edges are different.
https://github.com/sourcecred/sourcecred/blob/c9958e346ee789f41e1e406acc864c241245e70e/src/plugins/initiatives/declaration.js

Reply:
Sounds good.